examples/slurm/README

```diff
@@ -0,0 +1 @@
+This folder gives examples of setup to run i-PI on HPC systems.
```

Comment:
I am unsure about the usefulness of this full example. The slurm submission script has lots of useful bits, no doubt -- but is the whole tree useful?

Reply:
The idea was to have a truly complicated setup done from the beginning to the end, so that person can look ar everything together. It could be smaller in case of driver clients, but I didn't have anything with the driver that I could use for SL-RPC.

Reply:
ok! could we at least do a symbolic link for the CH.airebo file and for the data.*** files instead of recopying everything?

Reply:
Yes, I can in principle link all such files to `lammps/benz-graph-slc/`. I was thinking about making this only an "addon" to benz-graph-slc example, but decided that it will be unclear for newcomers compared to a standalone example. Linking also complicates things, but it's a reasonable compromise.

Reply:
yep just do it

Reply:
Done.